build.gradle

```diff
@@ -76,6 +76,7 @@ license {
                         "**/cycle.js",
                         "**/jquery*",
                         "**/sweetalert*",
+                        "**/datatables*",
```

Comment:
This is good -- but I don't see the `LICENSE-datatables.txt` file for datatables in the `licenses/` directory. We need to add it in there to complete this.